### PR TITLE
Issue #614 OSGi readiness

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

**Comment by ljacomet** [comment]
test this please

**Comment by ljacomet** [comment]
Damn Java 1.8 - looking into this

File: build.gradle [hunk 1]
```diff
@@ -266,6 +164,113 @@ subprojects {
...
+      developerConnection = 'scm:git:git@github.com:ehcache/ehcache3.git'
+    }
+    licenses {
+      license {
+        name = 'The Apache Software License, Version 2.0'
+        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+        distribution = 'repo'
+      }
+    }
+    developers {
```
Comment (chrisdennis) [inline build.gradle]: Not the topic of this issue, but since you touched this I'll bring it up.  What's policy on this list?

Comment (ljacomet) [inline build.gradle]: Need to get an answer for that and create an issue to clean up / complete for milestone 4

Comment (ljacomet) [inline build.gradle]: Create #644 for that

Comment (ljacomet) [inline build.gradle]: The reason I moved this block from the top to the bottom of the file is that it is code that needs little attention while all the basic version and plugins info is really more important and now shows properly at the top (copyright header excepted)

**Pull request merged**